cluster-provision/gocli/opts/cnao/cnao_test.go

```diff
@@ -51,7 +51,7 @@ var _ = Describe("CnaoOpt", func() {
 	})
 
 	It("should execute create CNAO without Multus", func() {
```

Comment:
Could you please add a test case for having CNAO deploy Multus v4 without dynamic-networks-controller?